ci/build_docs.sh

```diff
@@ -26,22 +26,23 @@ rapids-mamba-retry install \
   --channel "${PYTHON_CHANNEL}" \
   rmm librmm
 
-# Build CPP docs
-rapids-logger "Build Doxygen docs"
+export RAPIDS_VERSION_NUMBER=${VERSION_NUMBER}
```

Comment:
```suggestion
export RAPIDS_VERSION_NUMBER=23.08
```

the `VERSION_NUMBER` line can now be removed. it was only ever used in the `aws s3...` commands which are removed in your PR.

please remember to also update the corresponding entry in `update-version.sh`.